etc/config/triton.defaults.properties

```diff
@@ -0,0 +1,12 @@
+compilers=tritonHEAD
+defaultCompiler=tritonHEAD
+
+compiler.tritonHEAD.name=Triton (HEAD)
+compiler.tritonHEAD.exe=/home/simon/mambaforge/envs/tritonNightly/bin/python
```

Comment:
should be `/home/ce`?

Reply:
Defaults are whatever make sense as a default. I'm not any `/home/` is a sensible default, but I'm also not sure what else to suggest here. Typically we refer in the defaults to things like `/usr/bin/gcc` etc, where compilers are (probably) installed. It's expected users will override in their `*.local.properties`, and the `triton.amazon.properties` will be completely different (referring to something in `/opt/compiler-explorer/...` somewhere)

Reply:
I picked this default since I was hoping it makes it very obvious what it should be set to in the `*.local.properties`. I don't think it's common to have a package like Triton be installed globally using the system Python, but happy to take suggestions.

Reply:
I'd go with just `triton` or leave it unset entirely to make it error. But not a big deal really.